src/app/issues-viewer/card-view/card-view.component.html

```diff
@@ -29,8 +29,8 @@
           </mat-card-header>
           <mat-card-content>
             <mat-chip-list aria-label="Labels">
-              <span *ngIf="issue.milestone" class="octicon-milestone" octicon="milestone" color="grey" size="8"></span>
-              {{ issue.milestone?.title }}
+              <span *ngIf="issue.milestone.state" class="octicon-milestone" octicon="milestone" color="grey" size="8"></span>
+              {{ issue.milestone?.title !== 'Without a milestone' ? issue.milestone?.title : '' }}
```

Comment:
I think checking with `issue.milestone.state` is more efficient than checking with the string value of title. It is possible for users to intentionally set the milestone title as 'Without a milestone' but not possible for them to set the state to `null`

Reply:
Thanks for the suggestion!